debian/control

```diff
@@ -33,7 +33,9 @@ Vcs-Browser: https://github.com/pdudaemon/pkg-pdudaemon
 Package: pdudaemon
 Architecture: all
 Depends: ${misc:Depends},
-         ${python3:Depends}
+         ${python3:Depends},
+         python3-aiohttp,
+         python3-pymodbus,
```

Comment:
would it not be better to add to setup.py in that case?

Reply:
It makes no practical difference, but adding it here is easier since it requires no patching. I tried convincing dh-python in some other way, but haven’t found an easy one.

Reply:
Well it would make pdudaemon more consistent at least :) as install_requires is a subset of requiremnts.txt which seems odd

Reply:
> Well it would make pdudaemon more consistent at least :) as install_requires is a subset of requiremnts.txt which seems odd

Actually, requirements.txt is not used at all by anything but pip (?), and even pip AFAIR requires an explicit path to the requirements file.
But I agree that upstream this should be fixed properly (possibly also by dropping setup.py and replacing it with setup.cfg or pyproject.toml)

Reply:
Yeah pip is used for the docker file build; Could you submit a small MR to pdudaemon to get setup.py in sync then i can quickly merge it and fix this issue by packaging the new version instead :)